### PR TITLE
(node/auxtel-ill-control.cp) add el9 network config & tests

### PR DESCRIPTION
**Pull request by dtapiacl**
Redeploy auxtel-ill-control.cp.lsst.org as EL9 from EL7.

### CODE REVIEW

File: hieradata/node/auxtel-ill-control.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+nm::connections:
+  enp1s0:  #fqdn
```
Comment (jhoblitt) [inline hieradata/node/auxtel-ill-control.cp.lsst.org.yaml]:
```suggestion
  enp1s0:  # fqdn
```

**Review by jhoblitt — APPROVED** [review]
This looks good.  It is conventional to have a ` ` after the `#` starting a comment but it isn't a blocker.

File: hieradata/node/auxtel-ill-control.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/auxtel-ill-control.cp.lsst.org.yaml]: Thank you, i will update the code.

**Review by jhoblitt — APPROVED** [review]

**Review by gseriche — APPROVED** [review]
lgtm

**Pull request merged**